ts/deckoptions/DisplayOrder.svelte

```diff
@@ -35,44 +39,59 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
-    choices={newGatherPriorityChoices}
-    defaultValue={defaults.newCardGatherPriority}
-    bind:value={$config.newCardGatherPriority}
-/>
+<TitledContainer title={tr.deckConfigOrderingTitle()} {api}>
+    <Item>
+        <EnumSelectorRow
+            bind:value={$config.newCardGatherPriority}
+            defaultValue={defaults.newCardGatherPriority}
+            choices={newGatherPriorityChoices}
+            markdownTooltip={tr.deckConfigNewGatherPriorityTooltip()}
+        >
+            {tr.deckConfigNewGatherPriority()}
+        </EnumSelectorRow>
+    </Item>
 
-<EnumSelector
-    label={tr.deckConfigNewCardSortOrder()}
-    tooltip={tr.deckConfigNewCardSortOrderTooltip()}
-    choices={newSortOrderChoices}
-    defaultValue={defaults.newCardSortOrder}
-    bind:value={$config.newCardSortOrder}
-/>
+    <Item>
+        <EnumSelectorRow
+            bind:value={$config.newCardSortOrder}
+            defaultValue={defaults.newCardSortOrder}
+            choices={newSortOrderChoices}
+            markdownTooltip={tr.deckConfigNewCardSortOrderTooltip()}
+        >
+            {tr.deckConfigNewCardSortOrder()}
+        </EnumSelectorRow>
+    </Item>
 
-<EnumSelector
-    label={tr.deckConfigNewReviewPriority()}
-    tooltip={tr.deckConfigNewReviewPriorityTooltip()}
-    choices={reviewMixChoices()}
-    defaultValue={defaults.newMix}
-    bind:value={$config.newMix}
-/>
+    <Item>
+        <EnumSelectorRow
+            bind:value={$config.newMix}
+            defaultValue={defaults.newMix}
+            choices={reviewMixChoices()}
+            markdownTooltip={tr.deckConfigNewReviewPriorityTooltip()}
+        >
+            {tr.deckConfigNewReviewPriority()}
+        </EnumSelectorRow>
+    </Item>
 
-<EnumSelector
-    label={tr.deckConfigInterdayStepPriority()}
-    tooltip={tr.deckConfigInterdayStepPriorityTooltip()}
-    choices={reviewMixChoices()}
-    defaultValue={defaults.interdayLearningMix}
-    bind:value={$config.interdayLearningMix}
-/>
+    <Item>
+        <EnumSelectorRow
+            bind:value={$config.interdayLearningMix}
+            defaultValue={defaults.interdayLearningMix}
+            choices={reviewMixChoices()}
+            markdownTooltip={tr.deckConfigInterdayStepPriorityTooltip()}
+        >
+            {tr.deckConfigInterdayStepPriority()}
+        </EnumSelectorRow>
+    </Item>
 
-<EnumSelector
-    label={tr.deckConfigReviewSortOrder()}
-    tooltip={tr.deckConfigReviewSortOrderTooltip()}
-    choices={reviewOrderChoices}
-    defaultValue={defaults.reviewOrder}
-    bind:value={$config.reviewOrder}
-/>
+    <Item>
+        <EnumSelectorRow
+            bind:value={$config.reviewOrder}
+            defaultValue={defaults.reviewOrder}
+            choices={reviewOrderChoices}
+            markdownTooltip={tr.deckConfigReviewSortOrderTooltip()}
+        >
+            {tr.deckConfigReviewSortOrder()}
+        </EnumSelectorRow>
+    </Item>
+</TitledContainer>>
```

Comment:
Duplicate > sign after TitledContainer.

Reply:
I wonder why that is not detected by the formatters? Or what it is parsed as?

Reply:
Seems to be treated like the input of Anki's HTML editor or card templates, just as a literal character:

![image](https://user-images.githubusercontent.com/62722460/121771609-a5996000-cb70-11eb-89c8-ccaf6441a2e4.png)

Found it because I had 2021 scheduler enabled.